locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -2476,37 +2382,10 @@ protected String getFirstReachableAddress(NodeMetadata node, ConfigBag setup) {
         String result;
```

Comment:
Don't think the method `getFirstReachableAddress` is still referenced. Mark it deprecated.